brainscore_core/plugin_management/test_plugins.py

```diff
@@ -77,7 +77,7 @@ def _validate_environment_yml(self):
                 if len(python_specs) == 1:
                     python_spec = python_specs[0]
                     python_version = python_spec.split('python=')[1]
-                    assert python_version.startswith(('3.7', '3.8', '3.9'))  # should get these into a central place
+                    assert python_version.startswith(('3.11', '3.8', '3.9'))  # should get these into a central place
```

Comment:
I thought we only support >=3.11 now? I'm not even sure we need to be so stringent here. Could just duck-type and try if it works without explicitly checking the version. Maybe we can check the version and raise a warning but still try it.